src/ncp/ncp_base_mtd.cpp

```diff
@@ -2921,11 +2921,6 @@ template <> otError NcpBase::HandlePropertySet<SPINEL_PROP_STREAM_NET_INSECURE>(
     message = NULL;
```

Comment:
:+1:. With this change,  we can also remove the `message = NULL` and update the comment above it?
```suggestion
    
```